exercises/pangram/src/test/java/PangramsTest.java

```diff
@@ -8,37 +8,37 @@
 
 
     @Test
-    public void sentenceEmpty() {
+    public void emptySentenceIsNotPangram() {
         assertFalse(Pangrams.isPangram(""));
     }
 
     @Ignore
     @Test
-    public void pangramWithOnlyLowerCase() {
+    public void pangramWithOnlyLowerCaseLettersIsRecognizedAsPangram() {
         assertTrue(Pangrams.isPangram("the quick brown fox jumps over the lazy dog"));
     }
 
     @Ignore
     @Test
-    public void missingCharacterX() {
+    public void missingCharacterXIsNotPangram() {
         assertFalse(Pangrams.isPangram("a quick movement of the enemy will jeopardize five gunboats"));
     }
 
     @Ignore
     @Test
-    public void anotherMissingCharacterX() {
+    public void anotherMissingCharacterXIsNotPangram() {
```

Comment:
Same for this one. Maybe `anotherPhraseMissingCharacterXIsNotPangram` is more clear?